lib/erb_lint/cli.rb

```diff
@@ -41,7 +41,7 @@ def run(args = ARGV)
       if !@files.empty? && lint_files.empty?
         success!("no files found...\n")
       elsif lint_files.empty?
-        success!("no files given...\n#{option_parser}")
+        failure!("no files given...\n#{option_parser}")
```

Comment:
Do we think the other branch (discovery of files through the config) should fail too?

Reply:
Ah good point. I think we should also make it fail if there are no files found. 
@exterm any thoughts?

Reply:
Just for a reference point, `rubocop` doesn't actually fail if no files are found:

![Screen Shot 2020-01-13 at 2 18 42 PM](https://user-images.githubusercontent.com/833576/72284727-ae020a00-360f-11ea-866d-6bc5ec3278bc.png)

Similarly, if you get a non-ruby file, so it inspects nothing, it's still happy:

![Screen Shot 2020-01-13 at 2 30 23 PM](https://user-images.githubusercontent.com/833576/72285519-4c429f80-3611-11ea-8660-fcb492d79b2c.png)

So we're deviating from rubocop with this change. Just throwing this out for discussion.

Reply:
Mmmh, interesting - and good comparison. Any idea why rubocop would behave that way?

Reply:
Good question. Arguably, not finding files isn't particularly an error of the tool. As an analogy, would you want `[].map { ... }` raise an exception because there's nothing to map over?

What you _can_ do in rubocop, though, is `rubocop -L`, which lists all the files it will target. You can then write something like

```sh
if ! rubocop -L | grep '^' >/dev/null; then
  exit 1
fi
```

Not super pretty, I know...

Reply:
I'd just generally assume that running rubocop without files is a configuration error. It looks like it would do something but doesn't

Reply:
"fail early"!

We've had erblint running happily without any files for weeks in CI

Reply:
We can choose to make that a failure in erblint. I just checked and there's no way to make that a failure in rubocop at the moment. Worth opening an issue on their repo?

Reply:
@thegedge rubocop has default includes set up, doesn't it? Erblint just doesn't do anything if you don't give a list of files.

With that in mind, I think erblint should return an error status

Reply:
That works for me. Just wanted to call it out for discussion. I actually prefer errors in all cases. If we're not given files and we can't find any, I'd rather an error status.

Reply:
So the question is, do we want line 42 to also be `failure!`?

Reply:
absolutely